storybookjs/storybook
# Remove default CSS margin on preview body

This brings back the changes from #7742 which seem to have been removed in [4fc3b61](https://github.com/storybookjs/storybook/pull/7742/commits/4fc3b6187c54452971b53a57f8d23a8856f51c78)

Issue:

## What I did

Brought back `body` `margin: 0` changes. I ignored the following changes though, which I think were intentionally removed in #7750.

```
html, body {
  width: 100%;
  height: 100%;
}
```

## How to test

- Is this testable with Jest or Chromatic screenshots? Perhaps..
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? Don't think so

If your answer is yes to any of these, please make sure to include it in your PR.
